nautobot/core/utils/filtering.py

```diff
@@ -110,6 +110,7 @@ def get_filterset_parameter_form_field(model, parameter):
 
     # TODO(Culver): We are having to replace some widgets here because multivalue_field_factory that generates these isn't smart enough
     if isinstance(field, NumberFilter):
+        # TODO(Glenn): This is wrong for MultiValueDecimalFilter and MultiValueFloatFilter filters!
```

Comment:
What's the fix? More specific inheritance check?

Reply:
I think this falls under the #1483 umbrella - really our `BaseFilterSet` should generate valid forms on its own, obviating the need for this function to override the form fields on a case-by-case basis. #3214 is also related.

Reply:
Easy enough to fix for now though.